app.rb

```diff
@@ -2,5 +2,15 @@
 
 class App < Sinatra::Base
 
+  get '/newteam' do
+    erb :newteam
+  end
+
+
+  post '/team' do
+    @teamData = params
```

Comment:
You're not in Kansas anymore...
```suggestion
    @team_data = params
```